MovieQuiz/Services/StaticService.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+
+final class StatisticServiceImplementation: StatisticService {
+    
+    private let userDefaults = UserDefaults.standard
+    
+    private var correct: Int {
+        get {
+            userDefaults.integer(forKey: Keys.correct.rawValue)
+        }
+        set {
+            userDefaults.set(newValue, forKey: Keys.correct.rawValue)
+        }
+    }
+    
+    private var total: Int {
+        get {
+            userDefaults.integer(forKey: Keys.total.rawValue)
+        }
+        set {
+            userDefaults.set(newValue, forKey: Keys.total.rawValue)
+        }
+    }
+    
+    
+    
+    
+    var totalAccuracy: Double {
+        get {
+            return (Double(correct) / Double(total)) * 100
+        }
+    }
+    
+    var gamesCount: Int{
+        get {
+            userDefaults.integer(forKey: Keys.gamesCount.rawValue)
+        }
+        set {
+            userDefaults.set(newValue, forKey: Keys.gamesCount.rawValue)
+        }
+    }
+    
+    private(set)var bestGame: GameRecord {
```

Comment:
модификатор приватного сета пишут вот так `private(set) **пробел**  var`
```suggestion
    private(set) var bestGame: GameRecord {
```